roles/smallfile-bench/tests/inventory

```diff
@@ -0,0 +1,2 @@
+localhost
```

Comment:
what does localhost mean here?     Typically I use cluster loader to create a set of pods, and each pod is assigned a unique ID so that it can listen to the shared directory for test requests from a smallfile initiator.   The Dockerfile just starts the [launch.sh script](https://github.com/distributed-system-analysis/smallfile/blob/master/examples/docker/smallfile/launch.sh)  with the 2 specified environment variables, the script waits for a test to do, that's all a smallfile pod does by default.

Reply:
This script was added by ansible-galaxy command directly , I've no idea about this yet, how test cases will be executed here.  @jtaleric I might need help on this.

Reply:
You can remove the tests directory altogether.

Reply:
Boiler plate, can be removed.